docs/source/gswrap.rst

```diff
@@ -0,0 +1,6 @@
+*******
```

Comment:
Could you please double check that all the files which are added are needed (I see 28 changed files in PR)